lib/pair.ts

```diff
@@ -0,0 +1,268 @@
+import * as bitcoin from 'bitcoinjs-lib';
+import {encodeBtcAddress} from './encode';
+import {BigNumberish, BytesLike, BigNumber} from 'ethers';
+import {Obligation} from '../typechain/Obligation';
+import {Option} from '../typechain/Option';
+import {ITreasury} from '../typechain/ITreasury';
+import {OptionLib} from '../typechain/OptionLib';
+import {Ierc20} from '../typechain/Ierc20';
+import {Script} from './constants';
+import {BitcoinPriceFeed} from './feed';
+
+export type BtcAddress = {
```

Comment:
I would move the BTC types to a separate file. @gregdhill @danhper what do you think?